src/Files.App/ViewModels/UserControls/SidebarViewModel.cs

```diff
@@ -271,6 +273,19 @@ private Task<LocationItem> CreateItemHomeAsync()
 			return CreateSectionAsync(SectionType.Home);
 		}
 
+		private void CreateItemSettingsAsync()
```

Comment:
Is this still needed?

Reply:
Yes as long as I finish refactoring SidebarView it uses core item "INavigationControlItem" in SidebarItem.Item property though this is odd.